src/runtime/manifest-ast-types/manifest-ast-nodes.ts

```diff
@@ -623,41 +623,55 @@ export interface SchemaField extends BaseNode {
   name: string;
 }
 
+export enum SchemaFieldKind {
+  Primitive = 'schema-primitive',
+  KotlinPrimitive = 'kotlin-primitive',
+  Collection = 'schema-collection',
+  Reference = 'schema-reference',
+  OrderedList = 'schema-ordered-list',
+  Union = 'schema-union',
+  Tuple = 'schema-tuple',
+  Nested = 'schema-nested',
+  Inline = 'schema-inline',
+  InlineField = 'schema-inline-field',
+  TypeName = 'type-name' // same as inline.
```

Comment:
I don't understand this comment. It looks like it's different from inline?

Reply:
I agree, I believe it's the 'same' in that they are two different representations of the same thing (i.e. `TypeName` is an `Inline` type without specifying the exact fields involved).

The type checking code converts `TypeName`s to `Inline`s by looking up the top level definition of the schema.

Should I update the comment in a separate PR?

Reply:
If it's just this comment then updating it in this PR is fine.